Cdnfsd_Core.php

```diff
@@ -19,11 +19,10 @@ function get_engine() {
 			case 'cloudflare':
 				$engine_object = null;   // extension handles everything
 				break;
-
-				case 'transparentcdn':
```

Comment:
The changes in this file are not needed.  They may have been from an old copy.

Reply:
This file still has formatting changes.  Please revert to the current copy in master: https://raw.githubusercontent.com/W3EDGE/w3-total-cache/master/Cdnfsd_Core.php